activity_calendar/templates/activity_calendar/activity_moment_form_page.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'core/base.html' %}
+
+{% load static %}
+{% load render_form %}
+{% load generic_field %}
+{% load activity_default_display_tag %}
+
+{% block content-frame-class %}
+    wideContentFrame
+{% endblock %}
+
+{% block content %}
+    <h2>Change specific activity moment</h2>
+    <p>
+        Below you can edit attributes for this particular activity instance. Any field can be left empty. In that case
+        it will copy the attribute from the general activity instead.
+    </p>
+
```

Comment:
It would be nice to have the date/time of the activity(moment) somewhere as well